src/audio/voice_internal.c

```diff
@@ -0,0 +1,42 @@
+#include "global.h"
+
+#pragma GLOBAL_ASM("asm/non_matchings/code/voice_internal/D_801E0EC0.s")
+
+UNK_TYPE func_801A51F0(UNK_TYPE arg0) {
+    switch (arg0) {
+        case 1:
+        case 4:
+        case 5:
+        case 11:
+        case 13:
+        case 14:
+        case 15:
+            return -1;
+        default:
+            return 0;
+    }
+}
```

Comment:
The hell is this?

Reply:
This function has a jumptable which is bigger than the function asm itself. For some reason IDO didn't like that and it was giving problems to build it with a `#pragma GLOBAL_ASM`, so we took the simple route of just decomping it to avoid problems building